ros/src/waypoint_updater/waypoint_updater.py

```diff
@@ -52,7 +52,7 @@ def __init__(self, init_for_test=False):
             self.periodically_publish_waypoints()
 
     def periodically_publish_waypoints(self):
-        rate = rospy.Rate(50)
+        rate = rospy.Rate(20)
```

Comment:
Published waypoints go to waypoint follower (code from Autoware), which itself publishs with 30 Hz. Therefore I think we should not got below 30 Hz here, at least in the final code. For testing it's ok for now.